mirrord/cli/Dockerfile

```diff
@@ -26,12 +26,34 @@ RUN cargo +nightly-2025-08-01 zigbuild -p mirrord -p mirrord-layer -Z bindeps --
 RUN cp /build/target/$(cat /.platform)/release/mirrord /mirrord
 RUN cp /build/target/$(cat /.platform)/release/libmirrord_layer.so /libmirrord_layer.so
 
+# For ARM64 builds, also build x86_64 version to support cross-platform containers
+RUN if [ "$TARGETARCH" = "arm64" ]; then \
+    cargo +nightly-2025-08-01 zigbuild -p mirrord-layer -Z bindeps --target x86_64-unknown-linux-gnu --release --locked && \
```

Comment:
Also maybe make this symetrical, we should maybe bundle `arm64` with `amd64` target too no? just to cover if x86 computer runs an arm container

Reply:
I think this is too uncommon to care about for now